server/endpoints.go

```diff
@@ -46,21 +46,21 @@ const (
 	// Reports that are going to be returned are specified by list of cluster IDs that is part of request body
 	ReportForListOfClustersPayloadEndpoint = "organizations/{org_id}/clusters/reports"
 	// LikeRuleEndpoint likes rule with {rule_id} for {cluster} using current user(from auth header)
-	LikeRuleEndpoint = "clusters/{cluster}/rules/{rule_id}/error_key/{error_key}/users/{user_id}/like"
+	LikeRuleEndpoint = "clusters/{cluster}/rules/{rule_id}/error_key/{error_key}/organizations/{org_id}/users/{user_id}/like"
```

Comment:
I'd add this resource selector at the beginning, probably it will have more logic.

Reply:
I agree it would make more sense, but I just copied the structure from the ACK (system wide) endpoints, we can refactor all of them at once, it requires a small change in smart-proxy too, OK?

Reply:
Sure